src/Microsoft.ML.SearchSpace/Attribute/ChoiceAttribute.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+using System.Linq;
+using Microsoft.ML.SearchSpace.Option;
+
+namespace Microsoft.ML.SearchSpace
+{
+    [AttributeUsage(AttributeTargets.Property | AttributeTargets.Field, Inherited = false, AllowMultiple = false)]
+    public class ChoiceAttribute : Attribute
```

Comment:
`public sealed class ChoiceAttribute : Attribute`

Attributes should be sealed.

Reply:
### Resolved